packages/dev/core/src/Meshes/abstractMesh.ts

```diff
@@ -536,10 +542,41 @@ export abstract class AbstractMesh extends TransformNode implements IDisposable,
      */
     public alphaIndex = Number.MAX_VALUE;
 
+    /**
+     * If set to true, a mesh will only be visible only if its parent(s) are also visible (default is false)
+     */
+    public get inheritedVisibility(): boolean {
+        return this._internalAbstractMeshDataInfo._inheritedVisibility;
+    }
+
+    public set inheritedVisibility(value: boolean) {

```

Comment:
Naming feedback: I may think that inheritedVisibility is the visibility value that was inherited, not that it should inherit visibility or not.  Would inheritVisibility be more clear?

Reply:
Done! thanks for the suggestion :-)